pkg/controller/projects/deploytokens/controller.go

```diff
@@ -95,18 +97,20 @@ func (e *external) Observe(ctx context.Context, mg resource.Managed) (managed.Ex
 		return managed.ExternalObservation{ResourceExists: false}, nil
 	}
 
-	projectDeployTokenID, err := strconv.Atoi(externalName)
+	ID, err := strconv.Atoi(externalName)
```

Comment:
Local variables should always be lower case.

```suggestion
	id, err := strconv.Atoi(externalName)
```